### ISSUE TITLE
Add "Resume Run" feature.

### ISSUE THREAD

**Issue by Kigoli** [LiveSplit/LiveSplit#952]
This feature will save the paused time and use a splits file to resume the run using the completed splits as well as the paused time.

**Comment by Kigoli** [comment]
#953

**Comment by MinecraftPlaye** [comment]
@Kigoli any news on this one?

**Comment by Kigoli** [comment]
@MinecraftPlaye

I got it working, but it was definitely a hack job.

Because it wasn't up to the projects standards, it wasn't committed to the office project. 

This was a side project for me while I had some vacation time to burn.

If I ever feel inclined, I might pick it back up.

**Comment by Kigoli** [comment]
@MinecraftPlaye

I got it working, but it was definitely a hack job.

Because it wasn't up to the projects standards, it wasn't committed to the office project. 

This was a side project for me while I had some vacation time to burn.

If I ever feel inclined, I might pick it back up.

**Comment by gth** [comment]
Perhaps to get the feature implemented at all, remove the requirement for hibernate to support autosplitters?  i.e., the hibernate button would be disabled if an Autosplitter is detected.

**Comment by CryZe** [comment]
Yeah I believe for an initial implementation the auto splitters should be out of scope. So I guess I‘d be open to some prototypical implementation and we‘ll figure it out from there. Keep in mind that it probably will have to change a bit before we find a solution we are happy with, so be prepared for changes that are possibly required then.

**Comment by spenrose** [comment]
I am also trying to use LiveSplit to track progress over several days while leveling in Path of Exile. I see you have some code pushed now and would be happy to test it if you need some feedback/testing.

**Comment by DarkRTA** [comment]
We should also have an option to undo the pause in case this is somehow used as a preventive measure against LiveSplit crashing.

In this scenario we'd back up the currently active run after every split and if LiveSplit crashes for some reason, offer to restore that run upon the next launch.

Granted in most cases its possible to still finish a run even after the timer crashes.

**Comment by shane1595042264** [comment]
guys I really want this feature

**Comment by ThatGuyMike7** [comment]
same, need this for tracking time in casual Path of Exile runs

**Comment by Brambor** [comment]
My idea for this feature: Just save on closing. When launching, if you don't want the last run, press reset.

It should be default baked in like that.

**Comment by Shotnex4** [comment]
Hey everyone, I also missed this feature and tried to solve it myself.
I developed a LiveSplit Component which you can use to save your progress mid run to a file. With that you can close LiveSplit and even shutdown your PC. Just import the file when you are back and you can continue:
https://github.com/Shotnex4/LiveSplit.TimeAttackPause
